src/ensemble_scheduler/ensemble_scheduler.cc

```diff
@@ -1332,7 +1410,24 @@ EnsembleScheduler::Enqueue(std::unique_ptr<InferenceRequest>& request)
       TRITONSERVER_TRACE_TENSOR_QUEUE_INPUT, "EnsembleScheduler Enqueue");
 #endif  // TRITON_ENABLE_TRACING
 
+  std::unique_ptr<InferenceResponse> cached_response;
+  if (info_->is_cache_enabled_) {
+    CacheLookUp(request, cached_response);
+  }
+
+  if (cached_response != nullptr) {
+    LOG_VERBOSE(1)
+        << "Cache Hit: Top-level 'ensemble' request is found in cache";
```

Comment:
I think we can probably remove most of the verbose logging added here. Ideally the `LOG_VERBOSE(2)` at [top level](https://github.com/triton-inference-server/core/blob/56d60ccc750d5011fae084c359f4eaf3f94fb8fc/src/cache_manager.cc#L408) Insert() and Lookup() calls should be informative enough.